meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=1', 'cpp_std=c++14'],
+)
+
+ebml_conf = configure_file(output: 'ebml_export.h', configuration: {'EBML_DLL_API': ''})
+libebml_incdir = include_directories(['.'])
+
+install_headers(
+  'ebml/Debug.h',
+  'ebml/EbmlBinary.h',
+  'ebml/EbmlConfig.h',
+  'ebml/EbmlContexts.h',
+  'ebml/EbmlCrc32.h',
+  'ebml/EbmlDate.h',
+  'ebml/EbmlDummy.h',
+  'ebml/EbmlElement.h',
+  'ebml/EbmlEndian.h',
+  'ebml/EbmlFloat.h',
+  'ebml/EbmlHead.h',
+  'ebml/EbmlId.h',
+  'ebml/EbmlMaster.h',
+  'ebml/EbmlSInteger.h',
+  'ebml/EbmlStream.h',
+  'ebml/EbmlString.h',
+  'ebml/EbmlSubHead.h',
+  'ebml/EbmlTypes.h',
+  'ebml/EbmlUInteger.h',
+  'ebml/EbmlUnicodeString.h',
+  'ebml/EbmlVersion.h',
+  'ebml/EbmlVoid.h',
+  'ebml/IOCallback.h',
+  'ebml/MemIOCallback.h',
+  'ebml/MemReadIOCallback.h',
+  'ebml/SafeReadIOCallback.h',
+  'ebml/StdIOCallback.h',
+  ebml_conf,
+  subdir : 'ebml',
+)
+
+#build as static library for simplicity. shared not supported on MSVC.
+libebml = static_library('libebml',
+  'src/Debug.cpp',
+  'src/EbmlBinary.cpp',
+  'src/EbmlContexts.cpp',
+  'src/EbmlCrc32.cpp',
+  'src/EbmlDate.cpp',
+  'src/EbmlDummy.cpp',
+  'src/EbmlElement.cpp',
+  'src/EbmlFloat.cpp',
+  'src/EbmlHead.cpp',
+  'src/EbmlMaster.cpp',
+  'src/EbmlSInteger.cpp',
+  'src/EbmlStream.cpp',
+  'src/EbmlString.cpp',
+  'src/EbmlSubHead.cpp',
+  'src/EbmlUInteger.cpp',
+  'src/EbmlUnicodeString.cpp',
+  'src/EbmlVersion.cpp',
+  'src/EbmlVoid.cpp',
+  'src/IOCallback.cpp',
+  'src/MemIOCallback.cpp',
+  'src/MemReadIOCallback.cpp',
+  'src/SafeReadIOCallback.cpp',
+  'src/StdIOCallback.cpp',
+  include_directories : libebml_incdir,
+  install : true,
+)
+
+pkg = import('pkgconfig')
+pkg.generate(libebml)
```

Comment:
`EBML_STATIC_DEFINE` is not defined when the library is built as a static library.

Would it be possible to provide a CMake package information as well ? That's what current libebml users rely on.

Reply:
This is completely unused.